jormungandr-lib/src/interfaces/reward_parameters.rs

```diff
@@ -106,6 +141,11 @@ mod test {
                     epoch_start: Epoch::arbitrary(g),
                     epoch_rate: NonZeroU32::new(Arbitrary::arbitrary(g))
                         .unwrap_or(NonZeroU32::new(1).unwrap()),
+                    rewards_limit: Ratio::arbitrary(g),
+                    npools: NonZeroU32::new(Arbitrary::arbitrary(g))
+                        .unwrap_or(NonZeroU32::new(1000).unwrap()),
+                    npools_threshold: NonZeroU32::new(Arbitrary::arbitrary(g))
+                        .unwrap_or(NonZeroU32::new(100).unwrap()),
```

Comment:
why `100` ?

Reply:
See above.

Reply:
this is for the test. `1` is what has been used everywhere else to replace the `0` case.

Reply:
See above.